src/app/components/course-card/CourseCardComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Colors } from "../../common/assets";
+import { ICourseCardProps } from "./models";
+import {
+  Description,
+  Title,
+  Image,
+  Wrapper,
+  Content,
+  ParamsComponent,
+  Button,
+  DescriptionText,
+} from "./styles";
+
+export const CourseCardComponent: React.FC<ICourseCardProps> = ({
+  id,
+  title,
+  imageLink,
+  description,
+  lessonsCount,
+  skills,
+  rating,
+  onClickFunction,
+}) => {
+  return (
+    <Wrapper>
+      <Title>
+        <div style={{ margin: "5px 0px 0px 20px" }}>{title}</div>
+      </Title>
+      <Content>
+        <Image src={imageLink}></Image>
+        <Description>
+          <DescriptionText>{description}</DescriptionText>
+        </Description>
+        <ParamsComponent>
+          <div style={{ margin: "5px 0px 0px 20px" }}>
```

Comment:
The same goes here. Embedded styles are better to use when the styling needs to be dynamically generated and applied to a specific element or set of elements/variables. The code will be more structured and clean when styles are kept in one block.